Indicators/DeMarkerIndicator.cs

```diff
@@ -59,8 +59,8 @@ public DeMarkerIndicator(int period, MovingAverageType type = MovingAverageType.
         public DeMarkerIndicator(string name, int period, MovingAverageType type = MovingAverageType.Simple)
             : base(name)
         {
-            var _lastHigh = 0m;
-            var _lastLow = 0m;
+            _lastHigh = 0m;
+            _lastLow = 0m;
```

Comment:
Unrelated for posterity; I think we can remove these; also should be reset in the Reset method.